src/scenes/AcceleratorRouter/AcceleratorCohortNewView.tsx

```diff
@@ -0,0 +1,82 @@
+import { Typography, useTheme } from '@mui/material';
+import React, { useCallback, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { APP_PATHS } from 'src/constants';
+import AcceleratorMain from 'src/scenes/AcceleratorRouter/AcceleratorMain';
+import CohortForm from 'src/scenes/AcceleratorRouter/CohortForm';
+import CohortService from 'src/services/CohortService';
+import strings from 'src/strings';
+import { CreateCohortRequestPayload } from 'src/types/Cohort';
+import useForm from 'src/utils/useForm';
+import useSnackbar from 'src/utils/useSnackbar';
+
+type AcceleratorCohortNewViewProps = {
+  reloadData?: () => void;
+};
+
+export default function AcceleratorCohortNewView({ reloadData }: AcceleratorCohortNewViewProps): JSX.Element {
+  const history = useHistory();
+  const theme = useTheme();
+  const [isBusy, setIsBusy] = useState<boolean>(false);
+  const snackbar = useSnackbar();
+
+  const [record, setRecord] = useForm<CreateCohortRequestPayload>({
+    phase: 'Phase 0 - Due Diligence',
+    name: '',
+  });
+
+  const goToAcceleratorOverview = useCallback(() => {
+    history.push({ pathname: APP_PATHS.ACCELERATOR_OVERVIEW });
+  }, [history]);
+
+  const createNewCohort = useCallback(
+    async (cohort: CreateCohortRequestPayload) => {
+      // first create the cohort
+      let cohortId = -1;
+      setIsBusy(true);
+      const response = await CohortService.createCohort(cohort);
+      setIsBusy(false);
+      if (!response.requestSucceeded) {
+        snackbar.toastError();
+        return;
+      } else {
+        cohortId = response.data?.cohort?.id;
+      }
+
+      if (!cohortId) {
+        snackbar.toastError();
+        return;
+      }
+
+      // set snackbar with status
+      snackbar.toastSuccess(strings.formatString(strings.COHORT_ADDED, cohort.name) as string);
+
+      // navigate to accelerator overview
+      goToAcceleratorOverview();
```

Comment:
Pattern is to go to created object. `/accelerator/cohorts/<cohortId>`?
Do we have screens to view/edit an existing cohort?
For now an empty screen should work just fine (with maybe the breadcrumb back to the list of cohorts).

Reply:
It navigates back to the Overview screen here because that's what's in the AC for the ticket.

Reply:
Just wanted to call out what we do in the app, and most likely what we'd like to do here. @nickgraz

Reply:
We can follow up outside the scope of this PR for sure.

Reply:
Also since this is an MVP feature for internal use, don't want to increase scope. Please ignore my earlier comment.